### PR TITLE
Feat/306 cli config

### PR DESCRIPTION
**Pull request by lucapette**

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -4,6 +4,8 @@ load("@com_github_atlassian_bazel_tools//multirun:def.bzl", "multirun")
 load("@bazel_gazelle//:def.bzl", "gazelle")
 load("@io_bazel_rules_go//go:def.bzl", "TOOLS_NOGO", "nogo")
 
+# gazelle:proto disable_global
```
Comment (chrismatix) [inline BUILD:7]: To keep these in the same place, could you move this to L173?

Comment (lucapette) [reply BUILD:7]: sure thing!

File: infrastructure/cli/cmd/ui/ui.go [hunk 2]
```diff
@@ -9,21 +9,19 @@ import (
...
+	Short:            "Opens the Airy Core Platform UI in your local browser",
 	Long:             ``,
 	Run:              demo,
 }
 
 func demo(cmd *cobra.Command, args []string) {
 	// Initialize the api request
 
-	url := "http://chatplugin.airy/example.html"
+	url := "http://ui.airy/"
```
Comment (chrismatix) [inline infrastructure/cli/cmd/ui/ui.go:24]: I think this is still called `demo.airy` everywhere (docs, vagrant etc.)

**Review by chrismatix — APPROVED** [review]
Good stuff, left a small comment

**Pull request merged**